src/edu/wright/cs/raiderplanner/controller/CalendarController.java

```diff
@@ -140,6 +140,8 @@ private void setupAgendaButtons() {
 	 * This function will populate the agenda for the calendar.
 	 */
 	private void populateAgenda() {
+		if(calendarEvents.isEmpty()){return;}
```

Comment:
The formatting looks all wrong here.  You have the for loop at the same indentation level as the else and I am certain that our CheckStyle configuration does not permit the single line if construct that you have.